packages/@lwc/engine-core/src/framework/vm.ts

```diff
@@ -72,7 +72,6 @@ export enum RenderMode {
 
 export enum ShadowMode {
     Native,
-    None,
```

Comment:
Removed `ShadowMode.None` because `ShadowMode.Native` implies "not synthetic", making it a binary state. In other words, `RenderMode.Light` implies `ShadowMode.Native`.

Reply:
interesting way to put it, I agree!